mm2src/trezor/src/transport/webusb.rs

```diff
@@ -44,14 +45,14 @@ struct WebUsbLink {
 impl Link for WebUsbLink {
     async fn write_chunk(&mut self, chunk: Vec<u8>) -> TrezorResult<()> {
         if !self.device.is_open().await? {
-            self.reconnect().await?;
+            return MmError::err(TrezorError::DeviceDisconnected);
```

Comment:
does this mean device can no longer try to reconnect?

Reply:
Yes, exactly. The main reason is this issue https://github.com/KomodoPlatform/WebDEX/issues/672.
Once the device is disconnected, we no longer need to try to reconnect as the GUI should notify the user to reinitialize Hardware Wallet via a `Connect Hardware Wallet` button.

Reply:
Ok that makes sense thanks.